MapboxNavigation/RouteVoiceController.swift

```diff
@@ -205,11 +205,11 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate, AVAudioP
     }
     
     @objc open func didPassSpokenInstructionPoint(notification: NSNotification) {
-        guard !NavigationSettings.shared.voiceMuted else { return }
-        
         routeProgress = notification.userInfo![RouteControllerNotificationUserInfoKey.routeProgressKey] as? RouteProgress
         assert(routeProgress != nil, "routeProgress should not be nil.")
         
+        guard !NavigationSettings.shared.voiceMuted else { return }
```

Comment:
this can be undone.

Reply:
@bsudekum is there a reason why we have to undo this?